meson.build

```diff
@@ -37,16 +37,14 @@ if get_option('systemd')
 
   add_project_arguments ('-DHAVE_SYSTEMD_SD_DAEMON_H=1', language: 'c')
   add_project_arguments ('-DHAVE_SYSTEMD_SD_LOGIN_H=1', language: 'c')
-else
-  if get_option('offline_update')
-    error('Offline updates requires Systemd, use -Dsystemd=true to enable it or -Doffline_update=false to disable offline updates')
-  endif
+  add_project_arguments ('-DHAVE_SYSTEMD_SD_JOURNAL_H=1', language: 'c')
 endif
 
 elogind = []
 if get_option('elogind')
   elogind = dependency('elogind', version: '>=229.4')
   add_project_arguments ('-DHAVE_SYSTEMD_SD_LOGIN_H=1', language: 'c')
+  add_project_arguments ('-DHAVE_SYSTEMD_SD_JOURNAL_H=1', language: 'c')
```

Comment:
But elogind doesn't provide sd-journal, so why would we do that?

Reply:
Oh, it doesn't? I had no idea, I will fix that.

Reply:
Hmm, it seems it does? https://github.com/elogind/elogind/tree/main/src/libelogind/sd-journal

Reply:
![image](https://github.com/user-attachments/assets/874a7350-383d-49f0-9169-081a0bc1e5e4)